pubsub/rabbitmq/metadata.go

```diff
@@ -203,6 +207,14 @@ func createMetadata(pubSubMetadata pubsub.Metadata, log logger.Logger) (*metadat
 		}
 	}
 
+	if val, found := pubSubMetadata.Properties[metadataQueueType]; found && val != "" {
+		if queueTypeValid(val) {
```

Comment:
Can you lowercase "val" to be safer? So it would also accept "Classic"

Reply:
Sure. I added the lowercase.